plugins/woocommerce/client/blocks/assets/js/blocks/product-gallery/inner-blocks/product-gallery-large-image/edit.tsx

```diff
@@ -3,31 +3,45 @@
  */
 import { WC_BLOCKS_IMAGE_URL } from '@woocommerce/block-settings';
 import { useBlockProps, useInnerBlocksProps } from '@wordpress/block-editor';
+import { withProductDataContext } from '@woocommerce/shared-hocs';
+import { useProductDataContext } from '@woocommerce/shared-context';
 import { memo } from '@wordpress/element';
+import type { ReactElement } from 'react';
 import clsx from 'clsx';
 
 /**
  * Internal dependencies
  */
-import largeImageNextPreviousButtonMetadata from '../product-gallery-large-image-next-previous/block.json';
+import largeImageNextPreviousButtonMetadata from '../product-gallery-next-previous-buttons/block.json';
 import './editor.scss';
 
 const getInnerBlocksTemplate = () => [
 	[ largeImageNextPreviousButtonMetadata.name ],
 ];
 
-const Placeholder = memo( () => {
-	return (
-		<div className="wc-block-product-gallery-large-image wc-block-editor-product-gallery-large-image">
-			<img
-				src={ `${ WC_BLOCKS_IMAGE_URL }block-placeholders/product-image-gallery.svg` }
-				alt="Placeholder"
-			/>
-		</div>
-	);
-} );
+const ProductImage = memo(
```

Comment:
I don't understand why we are using `memo` here. This component appears to be pretty simple and will only rerender if the image changes which presumably is what we would want anyway? I can't see any performance issues unless I am missing something?

Reply:
I think I just left it, you're right!